src/app/shared/form/builder/ds-dynamic-form-ui/relation-lookup-modal/selection-tab/dynamic-lookup-relation-selection-tab.component.ts

```diff
@@ -29,10 +29,7 @@ import { createSuccessfulRemoteDataObject } from '../../../../../remote-data.uti
  * Tab for inside the lookup model that represents the currently selected relationships
  */
 export class DsDynamicLookupRelationSelectionTabComponent {
-  /**
-   * The label to use to display i18n messages (describing the type of relationship)
-   */
-  @Input() label: string;
+  @Input() relationshipType: string;
```

Comment:
Could you add a comment to describe this new Input, similar to how others are described?  It looks like a comment existed previously, but you accidentally removed it.

Reply:
Added TypeDoc